polyaxon/values.yaml

```diff
@@ -634,3 +634,13 @@ auth: {}
 istio: {}
 operators: {}
 ui: {}
+
+# priority for all pods created by this helm chart
+priorityClassName: ""
```

Comment:
This is duplicated in line [646](https://github.com/polyaxon/charts/pull/63/files#diff-ab0f09e6e4263eac169002565a284eff1deedc3c0d1e5504f8a0fdc99f30ba7aR646)